iceoryx_posh/source/runtime/service_discovery.cpp

```diff
@@ -35,7 +35,10 @@ ServiceContainer ServiceDiscovery::findService(const cxx::optional<capro::IdStri
     ServiceContainer searchResult;
     for (auto& service : tempSearchResult)
     {
-        searchResult.push_back(service.serviceDescription);
+        if (service.publisherCount > 0)
```

Comment:
Why is the `serverCount` not checked here? How can I find a `ServerPort`?

Reply:
You cannot for now, I talked without @elBoberido  and we need to figure out the best way to make this available in the discovery. For now it is just to not break existing functionality.